l10n_fi_business_code_validate/data/res_partner_id_category.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- Business ID -->
+    <record id="l10n_fi_business_code.res_partner_id_category_business_id" model="res.partner.id_category">
+        <field name="validation_code">self.validate_business_id(id_number)</field>
```

Comment:
Can you teach me what this does? I don't think I get it.

Reply:
This appends the validation method to the existing "business_id" -category created here
https://github.com/OCA/l10n-finland/pull/15/files#diff-9754556b0aaeaf06af60dfbc231d6d03

The `res.partner.id_category.validation_code`
https://github.com/OCA/partner-contact/blob/10.0/partner_identification/models/res_partner_id_category.py#L38

Is invoked when making changes to an id  number in the category
https://github.com/OCA/partner-contact/blob/10.0/partner_identification/models/res_partner_id_number.py#L19

And the Python code/method will be executed
https://github.com/OCA/partner-contact/blob/10.0/partner_identification/models/res_partner_id_category.py#L60

Reply:
Right, ok. I didn't understand it was that kind of code `validation_code` referred to :smile: